.eslintrc.js

```diff
@@ -85,7 +88,9 @@ module.exports = {
     ],
     "@typescript-eslint/no-non-null-assertion": "error",
     "@typescript-eslint/no-unnecessary-condition": "error",
-
+    // docusaurus does that
```

Comment:
The first one is ok, but `"react/jsx-uses-react"` seems it might be important. it would be good to revisit this at some point.